api/test/metrics/noop_sync_instrument_test.cc

```diff
@@ -9,42 +9,42 @@
 
 TEST(Counter, Add)
 {
-  std::shared_ptr<opentelemetry::metrics::Counter<long>> counter{
-      new opentelemetry::metrics::NoopCounter<long>("test", "none", "unitless")};
+  std::shared_ptr<opentelemetry::metrics::Counter<int64_t>> counter{
```

Comment:
uint64_t, as counter is always non-negative value.

Reply:
thanks, fixed